docs/endpoints/multi_user.rst

```diff
@@ -0,0 +1,909 @@
+job of the MEP is to start UEPs for users on request from the web service.  The actual
+processing of tasks is left to the individual UEPs.  This is accomplished through the
+well-known ``fork()`` |rarr| *drop privileges* |rarr| ``exec()`` Unix workflow,
+mimicking the approach of many other services (including Globus GridFTP and the Apache
+Web server).  In this manner, all of the standard Unix administrative user controls can
+be enforced.
+
+
+Configuration
+=============
+
+Configuration of a MEP starts with the ``--multi-user`` command line flag to the
+``configure`` subcommand:
+
+.. code-block:: console
+
+   # globus-compute-endpoint configure debug_queue --multi-user
+
+   Created multi-user profile for endpoint named <debug_queue>
+
+       Configuration file: /.../.globus_compute/debug_queue/config.yaml
+
+       Example identity mapping configuration: /.../.globus_compute/debug_queue/example_identity_mapping_config.json
+
+       User endpoint configuration template: /.../.globus_compute/debug_queue/user_config_template.yaml
+       User endpoint configuration schema: /.../.globus_compute/debug_queue/user_config_schema.json
+       User endpoint environment variables: /.../.globus_compute/debug_queue/user_environment.yaml
+
+   Use the `start` subcommand to run it:
+
+       # globus-compute-endpoint start debug_queue
+
+This command creates five configuration files, explained below.
+
+``config.yaml``
+---------------
+
+The default MEP ``config.yaml`` file is:
+
+.. code-block:: yaml
+
+   amqp_port: 443
+   display_name: null
+   identity_mapping_config_path: /.../.globus_compute/debug_queue/example_identity_mapping_config.json
+   multi_user: true
+
+The ``multi_user`` flag is required, but the ``identity_mapping_config_path`` is only
+required if the MEP process will have privileges to change users (e.g., if ``$USER =
+root``).  ``display_name`` is optional, but if set, determines how the MEP will appear
+in the `Web UI`_.  (And as the MEP does *not execute tasks*, :ref:`there is no engine
+block <cea_configuration>`.)
+
+``example_identity_mapping_config.json``
+----------------------------------------
+The default identity mapping configuration is valid, but will not work as it references
+``/bin/false`` and ``@example.com``.  This file shows what a valid configuration might
+look like, *but will require modification to work for an actual setup.*  (And hopefully
+the filename communicates the same idea!)  In particular, note that the mapping of
+identities from Globus Auth that are passed with each "start UEP" request is a key part
+of the MEP setup and security.
+
+.. warning::
+
+   At the Globus Compute web service, MEPs are, by default, open.  Any (Globus Auth
+   authenticated) user may submit tasks to them.  The intended and primary means of
+   access control is the identity mapping configuration.  For more information, see
+   `An Open Endpoint`_
```

Comment:
I think we should say that admins may use identity mapping, authentication policies, or a combination of the two to control access.

Reply:
I have to think about this.  The point was to discuss and encourage the identity mapping, given that the section is under the identity mapping file.  And by _default_, MEPs **are** open.